route/build/src/write_restart_pio.f90

```diff
@@ -983,7 +1095,7 @@ SUBROUTINE write_KWT_state(ierr, message1)
       select case(iVar)
         case(ixKWT%routed)
           allocate(array_3d_int(nSeg, nWave, nEns), stat=ierr, errmsg=cmessage)
-          if(ierr/=0)then; message1=trim(message1)//trim(cmessage)//':KWT routing state:'//trim(meta_mc(iVar)%varName); return; endif
+          if(ierr/=0)then; message1=trim(message1)//trim(cmessage)//':KWT routing state:'//trim(meta_kwt(iVar)%varName); return; endif
```

Comment:
Oh, doesn't the return still need to happen here?

Reply:
As well as the other instances below...

Reply:
just talked to Erik, he missed `return`. so no change